lib/beaker/host/unix/exec.rb

```diff
@@ -17,10 +23,10 @@ def reboot
       end
 
       # give the host a little time to shutdown
-      sleep 5 
+      sleep wait_time
```

Comment:
Being able to configure this is an improvement, but doesn't restore the previous functionality. This needs to retry if the uptime has changed, not just if uptime can be obtained.

Reply:
Unfortunately, this is correct. This still won't handle the case where the system hasn't rebooted, we need a combo of a `wait_time` and `retries` loop so that it's a multiplier to make this really work properly.

Each check will need to check and see if `uptime` is less than the previous uptime and fail after the nested loop time in the end.

Reply:
Understood, I'll work on getting this updated with support for retries today.